docker-compose.yaml

```
@@ -0,0 +1,14 @@
+services:
+  private-gpt:
+    build:
+      dockerfile: Dockerfile.local
+    volumes:
+      - ./local_data/:/home/worker/app/local_data
+      - ./models/:/home/worker/app/models
+    ports:
+      - 127.0.0.1:8080:8001
```

We chose the default 8001 for end users, swapping it to 8080 will just confuse people

```suggestion
      - 127.0.0.1:8001:8001
```

---

thanks for the explanation, I swapped the ports as you suggested